lib/nationbuilder/client.rb

```diff
@@ -56,6 +56,9 @@ def raw_call(path, method, body = {}, args = {})
       request_args[:query].merge!(body)
     else
       body[:access_token] = @api_key
+      unless body[:fire_webhooks].nil?
```

Comment:
Let's change this to `if body[:fire_webhooks]` so that it's easier to understand.